vnext/local-cli/generator-windows/templates/metro.config.js

```diff
@@ -15,12 +15,13 @@ const rnwPath = fs.realpathSync(
   path.resolve(require.resolve('react-native-windows/package.json'), '..'),
 );
 
+const rnInstallPath = fs.realpathSync('./react-native-installation');
```

Comment:
>react-native-installation [](start = 41, length = 25)

There's a potential for collisions here as I'm just picking a random directory name.  Open to suggestions.

Reply:
This logic here will also not always resolve to the same place as the current logic to create this folder does.

To match the current location you'd want.  `fs.realpathSync(path.resolve(require.resolve('react-native-windows/package.json'), '../../../react-native-installation'))`, or something like that, since you need to start from where react-native-windows was.

Reply:
Is there a place we can define the name in a global variable, or as the result of a global method, so we can just manage it from one location? The only reason I say method is we could auto-generate it based on the name of the app, so it won't collide in the future.